mobile/lib/features/wallet/balance_row.dart

```diff
@@ -18,28 +18,35 @@ class _BalanceRowState extends State<BalanceRow> with SingleTickerProviderStateM
   Widget build(BuildContext context) {
     WalletChangeNotifier walletChangeNotifier = context.watch<WalletChangeNotifier>();
 
+    final offchainBalance = walletChangeNotifier.offChain()?.formatted() ?? "n/a";
```

Comment:
👍 Better than a zero or nothing at all.

I wonder if we should eventually use skeletons during loading and just display an error pop-up if some key data cannot be loaded.

Reply:
> I wonder if we should eventually use skeletons during loading and just display an error pop-up if some key data cannot be loaded.

Can you elaborate? I am not sure I completely follow.

Reply:
I was just thinking about a nicer way to display information that is not yet available, and also information that we have failed to load.

In this case, showing a skeleton until the balance is available would be cool. But I'm not sure what would be the nicest way to display the fact that the balance could not be loaded. The label `n/a` is serviceable, but probably not the nicest.